src/mrack/transformers/beaker.py

```diff
@@ -66,6 +67,24 @@ def _get_distro_and_variant(self, host):
 
         return (required_distro, variant)
 
+    def _get_ks_meta(self, host):
+        """
+        Get `ks_meta` value from host or provisioning config or default if not defined.
+
+        The priority is following:
+            - host
+            - provisioning-config.yaml
+            - default (if nothing set)
+
+        """
+        return host.get(CONFIG_KEY, {}).get(
```

Comment:
Wouldn't `transformer._find_value` or some direct usage of `find_value_in_config_hierarchy` be more useful here? Especially if we use per-os config in provisioning config (comment below).

Reply:
let me try